docs/pages/admin-guides/management/guides/github-integration.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: GitHub Integration
```

Comment:
I would make the title more descriptive about what the integration entails, e.g., "Manage GitHub Access with Teleport" (depending on what's accurate here).

Reply:
not sure what's a good name here. i've changed it to `Git Access with GitHub` as we don't manage access yet. we only proxying git commands. the manage access part will come at a later time for another feature.

Reply:
For me, "Git Access with GitHub" implies that we're allowing users to access git using some feature of GitHub, which leaves out Teleport. Maybe "Proxy Git Commands with Teleport" would be descriptive enough for now?

Reply:
ya, that's better. Though i do think we should include GitHub in the title since it's really a github integration. What What do you think about "Proxy Git Commands with Teleport for GitHub"? Is it too long?

Reply:
I think that's good!